### PR TITLE
Update linux igpu run script

### PR DESCRIPTION
**Pull request by leonardozcm**

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run-igpu.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+source /opt/intel/oneapi/setvars.sh
+ 
+export USE_XETLA=OFF
+export SYCL_CACHE_PERSISTENT=1
+KERNEL_VERSION=$(uname -r)
+if [[ $KERNEL_VERSION != *"6.5"* ]]; then
+    export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
```
Comment (Oscilloscope98) [inline python/llm/dev/benchmark/all-in-one/run-igpu.sh:7]: I am not sure whether this `SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1` is useful for igpu on Linux. @MeouSker77 What do you think? :)

Comment (MeouSker77) [reply python/llm/dev/benchmark/all-in-one/run-igpu.sh:7]: not sure too

**Review by Oscilloscope98 — APPROVED** [review]
Others LGTM

File: python/llm/dev/benchmark/all-in-one/run-igpu.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+source /opt/intel/oneapi/setvars.sh
+ 
+export USE_XETLA=OFF
+export SYCL_CACHE_PERSISTENT=1
```
Comment (Oscilloscope98) [inline python/llm/dev/benchmark/all-in-one/run-igpu.sh:4]: Seems like we should have
```
export SYCL_CACHE_PERSISTENT=1
export BIGDL_LLM_XMX_DISABLED=1
```

image

**Pull request merged**